benchmark_test.go

```diff
@@ -0,0 +1,80 @@
+package waifu2x
+
+import (
+	"image"
+	"image/png"
+	"os"
+	"testing"
+)
+
+func Benchmark01(b *testing.B) {
+	const (
+		scale2xModelFile = "models/anime_style_art_rgb/scale2.0x_model.json"
+		noiseModelFile   = "models/anime_style_art_rgb/noise2_model.json"
+	)
+
+	fp, err := os.Open("testdata/neko_small.png")
+	if err != nil {
+		b.Fatalf("input file %v, %v", "testdata/neko.png", err)
+	}
+	defer fp.Close()
```

Comment:
__[gometalinter]__
* [errcheck] error return value not checked (defer fp.Close())